pkgs/development/tools/build-managers/rebar3/rebar3-nix-bootstrap

```diff
@@ -87,7 +87,7 @@ bootstrap_plugins(#data{plugins = Plugins}) ->
 -spec bootstrap_libs(#data{}) -> ok.
 bootstrap_libs(#data{erl_libs = ErlLibs}) ->
     io:format("Bootstrapping dependent libraries~n"),
-    Target = "_build/default/lib/",
+    Target = "_checkout/default/lib/",
```

Comment:
`_checkout/default/lib/$APP` is the wrong target path, it should be either `_checkouts/$APP` or `_build/default/lib/$APP`